.github/workflows/typespec-azure-integration-check.yml

```diff
@@ -0,0 +1,236 @@
+name: Azure TypeSpec Integration Check
+
```

Comment:
exclude the `packages/http-client-csharp`, `packages/http-client-java`, `packages/http-client-python` and `website` folder from triggering this CI